sos/plugins/apache.py

```diff
@@ -59,6 +61,10 @@ def setup(self):
 
         self.add_forbidden_path("/etc/httpd/conf/password.conf")
 
+        # If systemd is present, show status information.
+        if path.isdir('/run/systemd/system'):
+            self.add_cmd_output('systemctl status httpd')
```

Comment:
We're probably ok to just add the `add_cmd_output()` calls. If the command fails (or is missing), a_c_o already correctly handles that situation.